crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampInteractor.kt

```diff
@@ -298,44 +299,33 @@ internal class OnrampInteractor @Inject constructor(
             return OnrampCreateCryptoPaymentTokenResult.Failed(error)
         }
 
-        val platformPublishableKey = getOrFetchPlatformKey()
-        if (platformPublishableKey == null) {
-            val error = MissingPlatformSettingsException()
-            analyticsService?.track(
-                OnrampAnalyticsEvent.ErrorOccurred(
-                    operation = OnrampAnalyticsEvent.ErrorOccurred.Operation.CreateCryptoPaymentToken,
-                    error = error,
-                )
-            )
-            return OnrampCreateCryptoPaymentTokenResult.Failed(error)
-        }
-
-        return runCatching {
-            val apiKey = platformPublishableKey
-            when (val result = linkController.createPaymentMethodForOnramp(apiKey = apiKey)) {
-                is LinkController.CreatePaymentMethodResult.Success -> {
-                    result.paymentMethod
-                }
-                is LinkController.CreatePaymentMethodResult.Failed -> {
-                    throw result.error
+        // Get the platform publishable key
+        return getOrFetchPlatformKey()
```

Comment:
👀 Updated `getOrFetchPlatformKey()` to return `Result<>` instead of returning null on errors so we surface the error. You'll notice similar changes at the other call sites in this diff.